docs/WritingTests.asciidoc

```diff
@@ -1192,3 +1195,48 @@ the clone source job or specifying the variable explicitly:
 ----
 clone_job.pl --from https://openqa.opensuse.org  --host localhost 24 TESTDEBUG=1 SKIPTO=consoletest-yast2_i
 ----
+
+=== Defining a custom test schedule or custom test modules
+
+Normally the test schedule, that is which test modules should be executed and
+which order is prescribed by the +main.pm+ file within the test distribution.
```

Comment:
I think inserting a comma after order would make it more readable (not sure about actual grammar rules in English).